aggregate/housing_security/rent_stable_three_maintenance.py

```diff
@@ -82,37 +80,22 @@ def load_source_clean_data(indicator: str) -> pd.DataFrame:
     }
     data = pd.read_csv(**read_csv_arg)
     data.columns = [c.replace(".1", "") for c in data.columns]
-    data.columns = [
-        "SBA",
-        "PUMA",
-        "CD Name",
-        "N",
-        "SE",
-        "MOE\n(95% CI)",
-        "CV",
-        "Percent",
-        "Percent SE",
-        "Percent MOE\n(95% CI)",
-    ]
+    data.drop(columns=['SE', 'Percent SE'], inplace=True)
+    data.replace("%", "", regex=True, inplace=True)
+    data.replace(",", "", regex=True, inplace=True)
+    num_cols = ['N', 'MOE\n(95% CI)', 'CV', 'Percent', 'Percent MOE\n(95% CI)']
+    data[num_cols] = data[num_cols].apply(pd.to_numeric, errors="coerce")
 
     denom = pd.read_excel(**read_excel_arg)
     denom["PUMA"] = denom["PUMA"].astype(str)
-    denom.columns = [
-        "SBA",
-        "CD Name",
-        "PUMA",
-        "N",
-        "SE",
-        "MOE\n(95% CI)",
-        "CV",
-    ]
+    if 'Percent' in denom.columns:
+        denom.drop(columns=['SE', 'Percent', 'Percent SE', 'Percent MOE\n(95% CI)'], inplace=True)
+    else:
+        denom.drop(columns=["SE"], inplace=True)
 
     return data, denom
 
-
-def extract_geography_dataframe(
-    clean_data: pd.DataFrame, geography: str, denom=False
-) -> pd.DataFrame:
+def extract_geography_dataframe(clean_data: pd.DataFrame, geography: str) -> pd.DataFrame:
 
     if geography == "puma":
         clean_data["puma"] = clean_data["PUMA"].apply(func=clean_PUMAs)
```

Comment:
I would still like to see this moved to `load_source_clean_data` as it's a cleaning step

Reply:
this would have downstream issues where the extra geographies columns need to be dropped. Can't come up with a really elegant solutions for this is why I keep it this way.